diff --git a/node_modules/@starknet-react/core/dist/connectors/injected.js b/node_modules/@starknet-react/core/dist/connectors/injected.js
index f8ab8ce..7bf1687 100644
--- a/node_modules/@starknet-react/core/dist/connectors/injected.js
+++ b/node_modules/@starknet-react/core/dist/connectors/injected.js
@@ -2,7 +2,7 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.InjectedConnector = void 0;
 const get_starknet_1 = require("get-starknet");
-const index_1 = require("./index");
+const index_1 = require("./base");
 const errors_1 = require("../errors");
 class InjectedConnector extends index_1.Connector {
     constructor(options = {}) {
